src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -96,7 +110,17 @@ public Locomotive() {
 
         configuration = new LocomotiveConfig(testConfiguration, props);
 
-        Capabilities capabilities;
+        DesiredCapabilities capabilities;
+        
+        Capabilities extraCapabilities;
+		try {
+			extraCapabilities = configuration.capabilities().newInstance();
+		} catch (InstantiationException | IllegalAccessException e) {
+			e.printStackTrace();
+			logFatal(e.getMessage());
+			System.exit(1);
+			return;
```

Comment:
Why You are using return here, but do not add it in other places after System.exit() ?